### PR TITLE
WINUI-843320 Prepared the Customization UG content for Shimmer control in WinUI

### PR DESCRIPTION
**Pull request by GilbertmanojSF3971**
Prepared the Getting Started UG Customization content for Shimmer control in WinUI

### CODE REVIEW

File: winui/Shimmer/Customization.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+title: Customization in WinUI Shimmer control | Syncfusion
+description: Learn how to customize the appearance of the Syncfusion WinUI Shimmer(SfShimmer) control using various properties like Fill, Wave Color, Wave Width, Repeat Count and Wave Duration.
+platform: WinUI
+control: Shimmer
+documentation: ug
+---
+
+# Customization in WinUI Shimmer
+
+This section explains how to customize the appearance of the Syncfusion WinUI Shimmer (SfShimmer) control using various properties. You can modify the following aspects of the Shimmer effect:
```
Comment (Manivannan-E) [inline winui/Shimmer/Customization.md:12]: @GilbertmanojSF3971 This section explains how to customize the appearance of the Syncfusion WinUI Shimmer (SfShimmer) control using various properties. You will learn how to modify the following aspects of the shimmer effect:

File: winui/Shimmer/Customization.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Customization in WinUI Shimmer control | Syncfusion
+description: Learn how to customize the appearance of the Syncfusion WinUI Shimmer(SfShimmer) control using various properties like Fill, Wave Color, Wave Width, Repeat Count and Wave Duration.
+platform: WinUI
```
Comment (Manivannan-E) [inline winui/Shimmer/Customization.md:5]: @GilbertmanojSF3971 using various properties like Fill, WaveColor, WaveWidth, RepeatCount and WaveDuration.

File: winui/Shimmer/Customization.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+---
+
+# Customization in WinUI Shimmer
+
+This section explains how to customize the appearance of the Syncfusion WinUI Shimmer (SfShimmer) control using various properties. You can modify the following aspects of the Shimmer effect:
+
+## Fill
+
+The `Fill` property allows you to set the background color of the Shimmer view.
+
```
Comment (Manivannan-E) [inline winui/Shimmer/Customization.md:17]: @GilbertmanojSF3971 The Fill property allows you to specify the background color of the shimmer view.

File: winui/Shimmer/Customization.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+{% endhighlight %}
+{% endtabs %}
+
+![Fill customization in WinUI Shimmer](Shimmer_images/winui_shimmer_fill.gif)
+
+## Wave Color
+
+The `WaveColor` property lets you define the color of the shimmer wave.
+
```
Comment (Manivannan-E) [inline winui/Shimmer/Customization.md:38]: @GilbertmanojSF3971 The WaveColor property enables you to define the color of the shimmer wave.

File: winui/Shimmer/Customization.md (hunk 2 above)
Comment (GilbertmanojSF3971) [reply winui/Shimmer/Customization.md:5]: @Manivannan-E Sir Changed the content like using various properties like Fill, WaveColor, WaveWidth, RepeatCount and WaveDuration this.

File: winui/Shimmer/Customization.md (hunk 3 above)
Comment (GilbertmanojSF3971) [reply winui/Shimmer/Customization.md:17]: @Manivannan-E Sir changed the content to The Fill property allows you to specify the background color of the shimmer view.

File: winui/Shimmer/Customization.md (hunk 4 above)
Comment (GilbertmanojSF3971) [reply winui/Shimmer/Customization.md:38]: @Manivannan-E  Sir changed the content to The WaveColor property enables you to define the color of the shimmer wave in the PR.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**